api/src/auth/drivers/ldap.ts

```diff
@@ -136,7 +137,10 @@ export class LDAPAuthDriver extends AuthDriver {
 						const user = {
 							firstName: typeof object.givenName === 'object' ? object.givenName[0] : object.givenName,
 							lastName: typeof object.sn === 'object' ? object.sn[0] : object.sn,
-							email: typeof object.mail === 'object' ? object.mail[0] : object.mail,
+							email:
+								typeof object[mailAttribute ?? 'mail'] === 'object'
+									? object[mailAttribute ?? 'mail'][0]
+									: String(object[mailAttribute ?? 'mail']),
```

Comment:
Pls break `object[mailAttribute ?? 'mail']` out into a variable. Also, don't cast to string. Undefined is a valid response for this parameter.

Reply:
Alright :upside_down_face: